test/oPermutive.test.js

```diff
@@ -124,16 +124,12 @@ describe("OPermutive", () => {
 	describe("Consent is NOT set", () => {
 		beforeEach(() => {
 			fixtures.htmlCode('basic');
-			document.cookie = document.cookie + '=; expires=Thu, 01 Jan 1970 00:00:01 GMT;';
-			try {
-				oPermutive.init({
-					projectId: "1",
-					publicApiKey: "key"
-				});
-			} catch(e) {
-				proclaim.equal(e instanceof Error, true);
-				proclaim.equal(e.message, 'o-permutive: Could not initialise. No consent found');
-			}
+			document.cookie = 'foo=foo;';
+			oPermutive.init({
+				projectId: "1",
+				publicApiKey: "key",
+				consentFtCookie: true
```

Comment:
Doesn't `consentFTCookie: true` defeat the purpose of  `describe("Consent is NOT set",` ?

Reply:
good point

Reply:
no, but maybe it has caused some confusion in terms of the naming... as per the docs - setting consentFTcookie to true will mean that that o-permutive checks the consentCookie to derive the User's consent.

Reply:
maybe this should be changed - to something like "consent : getConsentFromCookie()"

Reply:
Yeah! It's a bit confusing

Reply:
> maybe this should be changed - to something like "consent : getConsentFromCookie()"

Do you mean passing a callback as argument? Sounds like a good idea.
Alternatively you could rename the property `consentFtCookie` to `checkForConsent` or `checkForConsentFtCookie`?

Reply:
> > maybe this should be changed - to something like "consent : getConsentFromCookie()"
> 
> Do you mean passing a callback as argument? Sounds like a good idea.
> Alternatively you could rename the property `consentFtCookie` to something like `checkForConsent` or `checkForConsentFtCookie`?

I like the second option as it also works well with passing config declaratively.